py/torch_tensorrt/dynamo/conversion/ops_evaluators.py

```diff
@@ -18,6 +18,7 @@
 )
 from torch_tensorrt.dynamo.conversion.impl.elementwise import sub, trunc_div
 from torch_tensorrt.fx.types import TRTTensor
+from torch_tensorrt.fx.utils import Frameworks, unified_dtype_converter
```

Comment:
We have our own types and switch from `unified_dtype_converter` to using `_enums.dtype.to` and `_enums.dtype.from` for type conversions. We shouldn't be using FX utils anymore.